lib/hooks.js

```diff
@@ -152,21 +109,13 @@ function onEndHookRunner (fn, request) {
   return fn(request.context)
 }
 
-function onGatewayReplaceSchemaHookRunner (fn, data) {
-  return fn(data.instance, data.schema)
-}
-
 module.exports = {
   Hooks,
   assignLifeCycleHooksToContext,
-  assignApplicationLifecycleHooksToContext,
   hooksRunner,
   preExecutionHooksRunner,
   hookRunner,
-  gatewayHookRunner,
-  preGatewayExecutionHooksRunner,
   preParsingHookRunner,
   onResolutionHookRunner,
-  onEndHookRunner,
-  onGatewayReplaceSchemaHookRunner
+  onEndHookRunner
```

Comment:
The functionality of Gateway hooks seems to be missing in the new package.

Reply:
It's there: https://github.com/mercurius-js/mercurius-gateway/blob/main/lib/handlers.js#L12

Reply:
https://github.com/mercurius-js/mercurius-gateway/blob/main/lib/handlers.js#L6 is pretty bad. I missed it when we did the extraction there, it should not do in-depth requirement and depend on an internal.

Moreover the hooks are not covered in the gateway docs.